app/models/index-card.ts

```diff
@@ -16,6 +16,21 @@ export interface LanguageText {
     '@value': string;
 }
 
+export enum ShareResourceTypes {
```

Comment:
nitpick: more accurate to credit OSFMAP for this list
```suggestion
export enum OsfmapResourceTypes {
```